src/mem/ruby/system/GPUCoalescer.cc

```diff
@@ -601,7 +601,6 @@ GPUCoalescer::hitCallback(CoalescedRequest* crequest,
                 case RubyRequestType_ATOMIC_NO_RETURN:
                     assert(pkt->isAtomicOp());
```

Comment:
Surprised there no compiler error about implicit fall-through here.  You could optionally add a `break;` here while you are touching this code